tinygrad/runtime/ops_gpu.py

```diff
@@ -9,8 +9,11 @@
 # see test/external/external_osx_profiling.py to determine this ratio. it's in like GPU clocks or something
 OSX_TIMING_RATIO = (125/3) if OSX else 1.0
 
+@functools.lru_cache(maxsize=1)
+def _get_errors(): return {getattr(cl, k): k for k in filter(lambda e: e.startswith("CL_"), dir(cl)) if getattr(cl, k) <=0}
+_get_errors()
```

Comment:
why not just `error_map = {getattr(cl, k): k for k in filter(lambda e: e.startswith("CL_"), dir(cl)) if getattr(cl, k) <= 0}`?

Reply:
@tyoc213 it's not resolved. I wrote you the same, no need to write anything that complex (with caches, functions, etc).

Tests failed. Also, I don't think that new file is the best place to put the test

Reply:
Got it now.

Fixed the test.

Moved the test to top dir and renamed it as it makes clear
 it is specific to ocl.